models/intermediate/salesforce_marketing_cloud__subscribers_enhanced.sql

```diff
@@ -1,4 +1,4 @@
--- this is meant to be stand alone since subscribers could belong to multiple lists, and I did not want to do a list agg to include this in the subscriber overview, but open for discussion
+-- this is meant to be stand alone since subscribers could belong to multiple lists, and I did not want to do a list agg to include this in the subscriber overview, but open for discussion 
```

Comment:
That comment was a leftover from a prior version. That comment was why I made salesforce_marketing_cloud__subscriber_lists so is now moot, so I have removed. I want to keep this as an int model since I added only a little information but is used in two downstreams.

Reply:
Nevermind my above comment. Let's keep the name you have since it will cause issues with postgres if we add the int naming convention.